.dockerignore

```diff
@@ -3,3 +3,4 @@
 .settings/
 */target/
 dspace/modules/*/target/
+Dockerfile.*
```

Comment:
```suggestion
Dockerfile.*

```
is there a preference to add a new line at the end of a file?

Reply:
There is no preference either way.